guides/v2.1/contributor-guide/backward-compatible-development/index.md

```diff
@@ -321,44 +321,47 @@ Review and refactor the class such that parts of the logic go into smaller speci
 Magento 2 must not have alternative APIs.
 Whenever you introduce a new implementation of some behavior, mark the old implementation as deprecated and specify the reason.
 
-### PHP, JS and XML
+### PHP, JS, and XML
```

Comment:
The added semicolon doesn't make sense here right?

Reply:
It's a comma, not a semicolon, and it's the grammatically correct way of separating items in a serial list.

Reply:
Ok, good to know, wasn't sure if the comma was there by mistake, but if this is the correct way I'm fine with it.